Packs/GoogleSafeBrowsing/Integrations/examples

```diff
@@ -1 +0,0 @@
-!url url="http://testsafebrowsing.appspot.com/apiv4/ANY_PLATFORM/MALWARE/URL/" using=GoogleSafeBrowsing_instance_1
```

Comment:
why delete this?

Reply:
the build thought it was an integration and got all muddled, i put it in its right place